src/modules/endpoint/buildResourceFilePath.js

```diff
@@ -1,10 +1,8 @@
 // @flow
 
-import fnv from 'fnv-plus'
 import { getResourceCacheFilesDirPath } from '../platform/constants/webview'
 import getExtension from './getExtension'
 
-export default (url: string, path: string, city: string) => {
-  const urlHash = fnv.hash(url).hex()
+export default (url: string, path: string, city: string, urlHash: string) => {
   return `${getResourceCacheFilesDirPath(city)}/${urlHash}.${getExtension(url)}`
```

Comment:
This does not correspond to the documentation in the docs (see persistence).
Also I think we don't need to have a urlHash in the resourceCache, since the actual hash is irrelevant (and only the filepath is the necessary information).

Reply:
Please adjust the docs.